src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1124,4 +1124,7 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
   <data name="Multiple_Nupkgs_Detected" xml:space="preserve">
     <value>Multiple nupkg files detected on '{0}' path to trust, only 1 is allowed.</value>
   </data>
+  <data name="VerifyCommand_NotSupported" xml:space="preserve">
+    <value>Package signature validation command is not supported on this platform.</value>
```

Comment:
Technically, verify is supposed to be able to do more than just signature validation, so this might not be very exact. 

I think we should consider changing the message accordingly. 

Given that nuget.exe on mono is not an extremely common scenario and this is a command that has existed for a very long time without much feedback about this, I'd be comfortable with keeping this message as is as well. 

This feedback is mostly about making sure we have all the scenarios considered.

Reply:
Forgot to mentioned that beyond this, the change looks great. 
I'd be comfortable approving once we confirm that this is something we have considered.

Reply:
Thanks for reviewing!
Yes, for now, we only have signature validation for verify command at
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyArgs.cs#L17-L22. And the signature validation could not be applied on Mono right now (we implement the signature validation in net472 code path by using many P/Invoke to Crypt32.dll, which only works on Windows platform).

I'm not sure what else validation options there will be in the future, and if there is any restrictions for those validation options on Mono. Any thoughts from the team? @NuGet/nuget-client Thanks!

Reply:
> I'm not sure what else validation options there will be in the future, and if there is any restrictions for those validation options on Mono

You are right, we can really predict. 
So the alternative here would be to *only* log the message when signing is requested.
Any future limitations can be added by a future implementer.

Reply:
Good point! Fixed accordingly.

Reply:
Didn't realize it was going to be that easy to make it more specific :D